### PR TITLE
DB-484: use CLIQZ theme from browser, not from extension

### PR DESCRIPTION
**Pull request by alver-cliqz**
For now theme start with browser, but will be replaced with theme from extension (it is the same). Started from browser version 50.0 extension will not replace theme (EX-3262)

### CODE REVIEW

**Comment by alver-cliqz** [comment]
retest this please

**Comment by alver-cliqz** [comment]
retest this please

**Comment by chrmod** [comment]
retest this please

**Comment by alver-cliqz** [comment]
Must be merged after EX-3262

File: magic_build_and_package.sh [hunk 1]
```diff
@@ -51,6 +51,21 @@ else
   export L10NBASEDIR=../l10n  # --with-l10n-base=...
 fi
 
+# process css for use theme in browser, not from extension
+echo '***** Copy css from extension *****'
+CLIQZ_EXT_URL='http://cdn2.cliqz.com/update/browser_beta/latest.xpi'
+if [ $CQZ_BUILD_ID ]; then
+  CLIQZ_EXT_URL='http://repository.cliqz.com/dist/'$CQZ_RELEASE_CHANNEL'/'$CQZ_VERSION'/'$CQZ_BUILD_ID'/cliqz@cliqz.com.xpi'
+fi
+wget -O cliqz@cliqz.com.xpi $CLIQZ_EXT_URL
```
Comment (chrmod) [inline magic_build_and_package.sh]: don't we do that already?

Comment (chrmod) [inline magic_build_and_package.sh]: should probably be integrated with rules.mk build script. or another way around

File: mozilla-release/browser/themes/linux/jar.mn [hunk 2]
```diff
@@ -51,6 +51,7 @@ browser.jar:
   skin/classic/browser/Toolbar-inverted.png
   skin/classic/browser/Toolbar-small.png
   skin/classic/browser/webRTC-indicator.css
+  skin/classic/browser/cliqz/theme.css                (cliqz/theme.css)
```
Comment (chrmod) [inline mozilla-release/browser/themes/linux/jar.mn]: how about putting it somewhere in `mozilla-release/browser/branding/cliqz/content` ? there is `aboutDialog.css there already

Comment (alver-cliqz) [inline mozilla-release/browser/themes/linux/jar.mn]: need a different file with same naming for different platforms

**Comment by alver-cliqz** [comment]
Please, take a look again. New way to use theme from extension in browser

**Comment by alver-cliqz** [comment]
retest this please

**Comment by alver-cliqz** [comment]
retest this please

**Comment by alver-cliqz** [comment]
retest this please

**Comment by alver-cliqz** [comment]
retest this please

**Pull request merged**